tfhe/src/shortint/oprf.rs

```diff
@@ -205,7 +252,7 @@ pub(crate) mod test {
     fn oprf_test_uniformity_ci_run_filter() {
         let sample_count: usize = 100_000;
 
-        let p_value_limit: f64 = 0.001;
+        let p_value_limit: f64 = 0.000_001;
```

Comment:
removing the keyswitch makes it work better ?

Reply:
No, it's unrelated

Reply:
ok, do you think the limit is better as it was or as it is now ?

Reply:
Better now as this test should fail with `p_value_limit` probability

Reply:
ok did you stress test it to check it did not fail in the CI ?

Reply:
This new bound is less strict (by a huge ratio) than the old one which IIRC only failed one on the CI (p=1/1000) so this should be fine

Reply:
got it, is it still significant enough ?

Reply:
Moved this change to https://github.com/zama-ai/tfhe-rs/pull/1179